src/bindings/python/requirements_test.txt

```diff
@@ -24,12 +24,15 @@ flake8_commas
 flake8_pep3101
 flake8_quotes
 mypy
-onnx==1.12.0
+onnx==1.12.0; platform_system != "Linux" and platform_machine != armv7l
+onnx==1.12.0; platform_system != "Linux" and platform_machine != aarch64
```

Comment:
looks like if we combine 2 lines, we can install onnx on any platform

Reply:
updated